src/main/java/redis/clients/jedis/netty/Connection.java

```diff
@@ -0,0 +1,80 @@
+package redis.clients.jedis.netty;
```

Comment:
I think we should not name this under .netty, but as .async or asyncio. The library implementation isn't important in the namespace,  IMHO - and we should do everything we can, to ensure it isn't.